src/views/LocationDetail.vue

```diff
@@ -62,13 +89,43 @@ export default {
       locationPriceTotal: null,
       locationPricePage: 0,
       loading: false,
+      // filter & order
+      priceFilter: '',
+      priceFilterList: constants.PRICE_FILTER_LIST,
+      priceOrder: constants.PRICE_ORDER_BY_LIST[1].key,
+      priceOrderList: constants.PRICE_ORDER_BY_LIST,
     }
   },
   mounted() {
+    this.priceFilter = this.$route.query[constants.FILTER_PARAM] || this.priceFilter
+    this.priceOrder = this.$route.query[constants.ORDER_BY_PARAM] || this.priceOrder
     this.getLocation(),
     this.getLocationPrices()
   },
+  computed: {
+    locationId() {
+      return this.$route.params.id
+    },
+    getCurrentPriceOrderIcon() {
+      let currentPriceOrder = this.priceOrderList.find(o => o.key === this.priceOrder)
+      return currentPriceOrder ? currentPriceOrder.icon : ''
+    },
+    getPricesParams() {
+      let defaultParams = { location_id: this.locationId, order_by: `${this.priceOrder}`, page: this.locationPricePage }
```

Comment:
```suggestion
      let defaultParams = { location_id: this.locationId, order_by: this.priceOrder, page: this.locationPricePage }
```
string interpolation not need, right ?

Reply:
Indeed, correcting it